pkg/kubectl/cmd/plugin.go

```diff
@@ -227,3 +228,15 @@ func isExecutable(fullPath string) (bool, error) {
 
 	return false, nil
 }
+
+func unique(items []string) []string {
```

Comment:
We could alternatively use the "sets" package from apimachinery: `"k8s.io/apimachinery/pkg/util/sets.NewString"`